portal/server/helpers/getUserRoles.js

```diff
@@ -1,6 +1,16 @@
 const { MAKER, CHECKER, PAYMENT_REPORT_OFFICER, ADMIN, READ_ONLY } = require('../constants/roles');
 
+const defaultResult = {
```

Comment:
This is a good addition, a minor thing but wonder if it's worth renaming to make it explicit that the default has no roles? Eg. `noRolesDefault`